lib/PayPal/Common/PayPalResourceModel.php

```diff
@@ -41,9 +41,11 @@ public function getLinks()
 
     public function getLink($rel)
     {
-        foreach ($this->links as $link) {
-            if ($link->getRel() == $rel) {
-                return $link->getHref();
+        if ($this->links) {
```

Comment:
Instead of just testing if this is a non-falsy value, can we add a test like `if (is_array($this->links))`?